### PR TITLE
#148 For CCM, cannot add test case for class Bar

### PR DESCRIPTION
**Pull request by uurl**
**Issue #148**

CCM.xsl:58-60

#148 For CCM, cannot add test case for class Bar because it is blocked by #114. Omission of the 'NAME' <op> for "getKey" messes up calculations. Add test case after #114 is fixed.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#256](https://github.com/yegor256/jpeek/issues/256) is now in scope, role is `REV`

File: src/main/java/org/jpeek/App.java [hunk 1]
```diff
@@ -72,10 +72,7 @@
  *  Once they are resolved, cover the metric with autotests and add it
  *  to reports list.
  *  (details on how to test the metrics are to be negotiated here - #107)
- * @todo #118:30min LCC metric has impediments (see puzzles in LCC.xml
- *  and in `MetricsTest`).
- *  Once they are resolved add it to reports list.
- */
+*/
 @SuppressWarnings
```
Comment (paulodamaso) [inline src/main/java/org/jpeek/App.java]: @uurl Why are you removing this puzzle? Puzzle #252 is still open.

**Comment by paulodamaso** [comment]
@rultor merge please

**Comment by uurl** [comment]
@paulodamaso I guess you must say "quality good"

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by paulodamaso** [comment]
@yegor256 ping

**Comment by yegor256** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@uurl Please fix the conflicts.

**Comment by uurl** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#256](https://github.com/yegor256/jpeek/issues/256); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#256](https://github.com/yegor256/jpeek/issues/256); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/jpeek#256` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#256](https://github.com/yegor256/jpeek/issues/256) is now out of scope

**Comment by paulodamaso** [comment]
@yegor256 I didn't received payment for this job yet; I've made the code revision and asked for merge 7 days before expiring date but got kicked anyways. Since 0crat kicked me when my job was already done it won't pay me ever, it'll be done manually.

**Comment by paulodamaso** [comment]
@yegor256 ping

**Comment by yegor256** [comment]
@0crat extra @paulodamaso 15

**Comment by 0crat** [comment]
> @0crat extra @paulodamaso 15 ([here](https://github.com/yegor256/jpeek/issues/256#issuecomment-401627275))

@yegor256 Argument "@paulodamaso" doesn't match regex `\d+min`

**Comment by paulodamaso** [comment]
@yegor256  Didn't worked, please take a look at this

**Pull request merged**